examples/simpleInvoke/main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"os"
+
+	"github.com/NethermindEth/juno/core/felt"
+	starknetgo "github.com/NethermindEth/starknet.go"
+	"github.com/NethermindEth/starknet.go/account"
+	"github.com/NethermindEth/starknet.go/rpc"
+	"github.com/NethermindEth/starknet.go/utils"
+	ethrpc "github.com/ethereum/go-ethereum/rpc"
+	"github.com/joho/godotenv"
+)
+
+var (
+	name           string = "testnet"
+	account_addr   string = "0x06f36e8a0fc06518125bbb1c63553e8a7d8597d437f9d56d891b8c7d3c977716"
+	privateKey     string = "0x0687bf84896ee63f52d69e6de1b41492abeadc0dc3cb7bd351d0a52116915937"
+	public_key     string = "0x58b0824ee8480133cad03533c8930eda6888b3c5170db2f6e4f51b519141963"
+	someContract   string = "0x4c1337d55351eac9a0b74f3b8f0d3928e2bb781e5084686a892e66d49d510d"
+	contractMethod string = "increase_value"
+)
+
+func main() {
+	godotenv.Load(fmt.Sprintf(".env.%s", name))
+	base := os.Getenv("INTEGRATION_BASE") //please modify the .env.testnet and replace the INTEGRATION_BASE with an starknet goerli RPC.
+	fmt.Println("Starting simpeCall example")
+	c, err := ethrpc.DialContext(context.Background(), base)
+	if err != nil {
+		fmt.Println("Failed to connect to the client, did you specify the url in the .env.mainnet?")
+		panic(err)
+	}
+	clientv02 := rpc.NewProvider(c)
+	account_address, err := utils.HexToFelt(account_addr)
+	if err != nil {
+		panic(err.Error())
+	}
+	ks := starknetgo.NewMemKeystore()
+	fakePrivKeyBI, _ := new(big.Int).SetString(privateKey, 0)
+
+	ks.Put(public_key, fakePrivKeyBI)
+
+	fmt.Println("Established connection with the client")
+
+	maxfee, err := utils.HexToFelt("0x9184e72a000")
+	if err != nil {
+		panic(err.Error())
+	}
+
+	InvokeTx := rpc.InvokeTxnV1{
+		MaxFee:        maxfee,
+		Version:       rpc.TransactionV1,
+		Nonce:         new(felt.Felt).SetUint64(108), // You will have to modify this accordingly
```

Comment:
Account has a nonce method that we should probably use instead of setting the nonce by hand

Reply:
I have updated it, instead of setting it manually, nonce is fetched through the account's nonce method